src/graph/optimizer/rule/CollapseProjectRule.cpp

```diff
@@ -11,6 +11,8 @@
 #include "graph/planner/plan/Query.h"
 #include "graph/util/ExpressionUtils.h"
 
+DEFINE_bool(enable_opt_collapse_project_rule, true, "");
```

Comment:
Why disable this rule?

Reply:
It's not disabled, it's just a switch. Actually, all opt rules need a switch.

Reply:
RBO should not be designed to be a tradeoff. Why need a switch?

Reply:
1. For test or debug.
2. RBO rule doesn't always make the plan better.